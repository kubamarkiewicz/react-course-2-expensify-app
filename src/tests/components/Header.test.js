import React from 'react';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import { shallow } from 'enzyme';
import Header from '../../components/Header.js';


/*test('should render header', () => {
	const renderer = new ReactShallowRenderer();
	renderer.render(<Header />);
	expect(renderer.getRenderOutput()).toMatchSnapshot();
});*/

test('should render header', () => {
	const wrapper = shallow(<Header />);
	expect(wrapper).toMatchSnapshot();
});