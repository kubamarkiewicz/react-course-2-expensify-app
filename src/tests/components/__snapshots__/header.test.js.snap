// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header 1`] = `
<header>
  <h1>
    Expensify Kubas
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/create"
  >
    Create
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
